index.js

```diff
@@ -331,6 +331,15 @@ export default class WorkspacesPlugin extends Plugin {
         for (let dependency in dependencies) {
           if (workspaces.find((w) => w.name === dependency)) {
             const existingVersion = dependencies[dependency];
+
+            /**
+             * If pnpm is being used, the Workspace protocol may also be used
+             * https://pnpm.io/workspaces#workspace-protocol-workspace
+             * if it is, these version references are handled on publish
+             * by `pnpm publish`.
+             */
+            if (existingVersion.startsWith('workspace:')) continue;
```

Comment:
I don't think this is correct. We need to make `_buildReplacementDepencencyVersion` do the right thing for `workspace:` prefixes.

Tests cases to add below

https://github.com/release-it-plugins/workspaces/blob/49fe6560bd24160666eac58f1b2fc812e2eadb25/tests/plugin-test.js#L1122

```js
    updatesTo({ existing: 'workspace:^1.0.0', new: '2.0.0', expected: 'workspace:^2.0.0' });
    updatesTo({ existing: 'workspace:~1.0.0', new: '2.0.0', expected: 'workspace:~2.0.0' });
    updatesTo({ existing: 'workspace:1.0.0', new: '2.0.0', expected: 'workspace:2.0.0' });
    updatesTo({ existing: 'workspace:^', new: '2.0.0', expected: 'workspace:^' });
    updatesTo({ existing: 'workspace:~', new: '2.0.0', expected: 'workspace:~' });
    updatesTo({ existing: 'workspace:*', new: '2.0.0', expected: 'workspace:*' });
```

Reply:
I don't really agree? -- pnpm does this for you, and you can't use the workspace protocol outside of pnpm.
re-implementing the details of how the workspace protocol works seems like it's beyond the responsibilities of relese-it-plugins/workspaces, and release-it in general

Reply:
it looks like when we do run release-it, _if_ there are any changes to the versions of anything, we need to run `pnpm i` before commit/tag, else ci-based workflows fail (due to out-dated lockfile). :thinking: 

where would be the best place to add that when pnpm is detected?

Reply:
maybe this would be better: https://github.com/release-it-plugins/workspaces/blob/master/index.js#L277

Reply:
added!